tests/testImageDecorrelation.py

```diff
@@ -245,19 +265,19 @@ def _testImages(self):
 
         # Expected (ideal) variance of difference image
         expected_var = self.svar + self.tvar
-        print('Expected variance:', expected_var)
+        print('EXPECTED VARIANCE:', expected_var)
```

Comment:
As long as I'm still developing these tasks, I find it reassuring to look at the numbers and make sure they make sense. I cannot explain it better than that. I count 1422 'print' statements in unit tests across the lsst stack.